From: Chad Monroe <chad@monroe.io>
Date: Sun, 26 Jul 2026 20:29:06 -0700
Subject: [PATCH] wifi: mt76: mt7915: disable firmware coredump generation

Walking the MCU memory regions for a coredump after an exception
can itself crash the host while the chip is in an undefined state.
Bail out of mt7915_coredump_build() for now until the dump walk is
made safe.

Fixes: SDG-2422

Signed-off-by: Chad Monroe <chad@monroe.io>
---
 mt7915/coredump.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/mt7915/coredump.c
+++ b/mt7915/coredump.c
@@ -308,6 +308,8 @@ static struct mt7915_coredump *mt7915_co
 	unsigned char *buf;
 	bool exception;
 
+	return NULL;
+
 	len = hdr_len;
 
 	if (coredump_memdump && crash_data->memdump_buf_len)
